cfme/tests/cloud/test_providers.py

```diff
@@ -353,6 +354,68 @@ def test_api_port_max_character_validation_cloud(appliance):
         assert text == prov.default_endpoint.api_port[0:15]
 
 
+@pytest.mark.tier(2)
+@pytest.mark.uncollectif(lambda provider: not provider.one_of(AzureProvider))
+def test_azure_subscription_required(request, provider):
+    """
+    Tests that provider can't be added w/o subscription
+
+    Metadata:
+        test_flag: crud
+    """
+    provider.subscription_id = ''
+    request.addfinalizer(provider.delete_if_exists)
+    flash = 'Credential validation was not successful: ' \
```

Comment:
Required: Use brackets here instead of \
('Credential validation was not successful: '
 'Incorrect credentials - check your Azure Subscription ID')